docs/GoGetSllDoc.md

```diff
@@ -0,0 +1,669 @@
+
+<h3>Version information</h3>
+
+<p>Connector Version: 1.0.0</p>
+
+<p>Authored By: Fortinet</p>
+
+<p>Certified: No</p>
+
+<h2>Installing the connector</h2>
+
+<p>Use the <strong>Content Hub</strong> to install the connector. For the detailed procedure to install a connector, click <a href="https://docs.fortinet.com/document/fortisoar/0.0.0/installing-a-connector/1/installing-a-connector" target="_top">here</a>.</p><p>You can also use the <code>yum</code> command as a root user to install the connector:</p>
+
+<pre>yum install cyops-connector-gogetssl</pre>
+
+<h2>Prerequisites to configuring the connector</h2>
+
+<ul>
+<li>You must have the credentials of GOGETSSL server to which you will connect and perform automated operations.</li>
+<li>The FortiSOAR&trade; server should have outbound connectivity to port 443 on the GOGETSSL server.</li>
+</ul>
+
+<h2>Minimum Permissions Required</h2>
+
+<ul>
+<li>Not applicable</li>
+</ul>
+
+<h2>Configuring the connector</h2>
+
+<p>For the procedure to configure a connector, click <a href="https://docs.fortinet.com/document/fortisoar/0.0.0/configuring-a-connector/1/configuring-a-connector">here</a></p>
+
+<h3>Configuration parameters</h3>
+
+<p>In FortiSOAR&trade;, on the Connectors page, click the <strong>GOGETSSL</strong> connector row (if you are in the <strong>Grid</strong> view on the Connectors page) and in the <strong>Configurations</strong> tab enter the required configuration details:</p>
+
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>Server URL</td><td>Specify the Rest API endpoint URL of the GOGETSSL server to connect and perform automated operations.</td></tr>
+<tr><td>API Key</td><td>Specify the API key to access the GOGETSSL Rest API endpoint to which you will connect and perform the automated operations.</td></tr>
+<tr><td>Verify SSL</td><td>Specifies whether the SSL certificate for the server is to be verified. <br/>By default, this option is selected, i.e., set to <code>true</code>.</td></tr>
+</tbody></table>
+
+<h2>Actions supported by the connector</h2>
+
+<p>You can use the following automated operations in playbooks and also use the annotations to access operations:</p>
+
+<table border=1><thead><tr><th>Function</th><th>Description</th><th>Annotation and Category</th></tr></thead><tbody><tr><td>Get Domain Alternative</td><td>Retrieve information about alternative domain names associated with a specific SSL/TLS certificate.</td><td>get_domain_alternative <br/>Investigation</td></tr>
+<tr><td>Get Domain Emails</td><td>Retrieve email addresses associated with a specific domain.</td><td>get_domain_emails <br/>Investigation</td></tr>
+<tr><td>Get Domain Emails for Geo Trust</td><td>Retrieve email addresses associated with a domain specifically for GeoTrust SSL certificates.</td><td>get_domain_emails_for_geo_trust <br/>Investigation</td></tr>
```

Comment:
Update the action name and annotation + Action description

e.g. Action name: Get Geotrust Approval Emails
Description: Retrieve an array of valid Geotrust approver email addresses for specified domain.